cosmwasm/deploy-contract.js

```diff
@@ -36,7 +46,11 @@ const makeServiceRegistryInstantiateMsg = ({ governanceAccount }) => {
     return { governance_account: governanceAccount };
 };
 
-const makeMultisigInstantiateMsg = ({ governanceAddress, blockExpiry }, { Rewards: { address: rewardsAddress } }) => {
+const makeMultisigInstantiateMsg = ({ adminAddress, governanceAddress, blockExpiry }, { Rewards: { address: rewardsAddress } }) => {
+    if (!validateAddress(adminAddress)) {
```

Comment:
you can simplify validation of multiple params via this approach
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/gas-service.js#L252

Reply:
in this particular case `validateParameters` cannot be used because it is unable to validate a cosmos address. And since the function is part of the evm package, I don't think it makes sense to extend it for cosmos